### ISSUE TITLE
Competing Consumers Support

### ISSUE THREAD

**Issue by idoshamun** [bmavity/ges-client#61]
Competing Consumer is a new type of subscription released in version 3.2.
Is it possible to add support for it?

http://docs.geteventstore.com/introduction/competing-consumers/

**Comment by bmavity** [comment]
Yes, it will happen at some point. I am just swamped with multiple projects right now and can't get to it.

I will keep the issue open to act as a reminder for me to devote some time.

**Comment by CharlieBrown222** [comment]
Hello Guys, 
any news about this topic?
thank you so much!!

**Comment by rsciarretta** [comment]
Hello,
I'm interested at this implementation too.
Is there any roadmap?

Btw...your client is very cool!!! Congrats!!

**Comment by gregoryyoung** [comment]
I am interested in the state as well.

I am guessing that the branch
https://github.com/bmavity/ges-client/tree/dev/tcp is the one worth
watching?

Greg

On Sat, May 14, 2016 at 11:42 AM, Roberto Sciarretta <
notifications@github.com> wrote:

> Hello,
> I'm interested at this implementation too.
> Is there any roadmap?
> 
> Btw...your client is very cool!!! Congrats!!
> 
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly or view it on GitHub
> https://github.com/bmavity/ges-client/issues/61#issuecomment-219211064

## 

Studying for the Turing test

**Comment by bmavity** [comment]
Hi All,

I have rid myself of my extra projects, so I should have some free time soon. The branch that is about to be merged is the https://github.com/bmavity/ges-client/tree/connection_events branch. It has more robust retries and fixes to some issues with internals.

Also @gregoryyoung , I emailed you in response to your inquiry a couple of weeks ago. Did you receive the email?

**Comment by gregoryyoung** [comment]
Yep I did but have been travelling the last few weeks so have not had a
chance to look through things yet.

On Sat, May 14, 2016 at 6:22 PM, Brian Mavity notifications@github.com
wrote:

> Hi All,
> 
> I have rid myself of my extra projects, so I should have some free time
> soon. The branch that is about to be merged is the
> https://github.com/bmavity/ges-client/tree/connection_events branch. It
> has more robust retries and fixes to some issues with internals.
> 
> Also @gregoryyoung https://github.com/gregoryyoung , I emailed you in
> response to your inquiry a couple of weeks ago. Did you receive the email?
> 
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> https://github.com/bmavity/ges-client/issues/61#issuecomment-219229183

## 

Studying for the Turing test
